tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -40,7 +45,14 @@ import {TbdevUploadDialogComponent} from './tbdev_upload_dialog_component';
   ],
 })
 export class TbdevUploadButtonComponent {
-  constructor(private dialog: MatDialog) {}
+  @HostBinding('class.shown') shown: boolean = false;
```

Comment:
Since the `shown` is initialized at the constructor, we can remove the value here.

Reply:
Done.